tests/test_job.py

```diff
@@ -90,8 +90,13 @@ def __init__(self, gitlab_url=None):
         )
         api.add_transition(
            GET('/projects/1234/repository/branches/useless_new_feature'),
+           Ok({'commit': _commit(id=rewritten_sha, status='running')}),
+            from_state='pushed',
+        )
+        api.add_transition(
+           GET('/projects/1234/repository/branches/useless_new_feature'),
            Ok({'commit': _commit(id=rewritten_sha, status='success')}),
-           from_state='passed'
+            from_state='passed'
```

Comment:
@aschmolck Extra space before `from_state` (also in the transition above)

Reply:
Weird, that was due to a bug in emacs python mode indentation logic!